Traceback (most recent call last):
  File "stationTrendsByYear.py", line 73, in <module>
    calcAnnualStationStats(beginYear, endYear)
  File "stationTrendsByYear.py", line 35, in calcAnnualStationStats
    "Direction","Speed","CoverageCode", "Depth1","Depth6"])
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 655, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 405, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 764, in __init__
    self._make_engine(self.engine)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 995, in _make_engine
    self._engine = klass(self.f, **self.options)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 1997, in __init__
    self.columns, self.num_original_columns = self._infer_columns()
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 2372, in _infer_columns
    line = self._buffered_line()
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 2443, in _buffered_line
    return self._next_line()
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 2548, in _next_line
    orig_line = self._next_iter_line(row_num=self.pos + 1)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 2608, in _next_iter_line
    return next(self.data)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/site-packages/pandas/io/parsers.py", line 2132, in _read
    line = f.readline()
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/gzip.py", line 289, in read1
    return self._buffer.read1(size)
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/gzip.py", line 463, in read
    if not self._read_gzip_header():
  File "/users/a/a/aametcal/anaconda3/lib/python3.6/gzip.py", line 411, in _read_gzip_header
    raise OSError('Not a gzipped file (%r)' % magic)
OSError: Not a gzipped file (b'19')
